extensions/wikia/UserLogin/css/UserLoginDropdown.globalNavigation.scss

```diff
@@ -54,7 +54,7 @@
 	}
 
 	.WikiaForm {
-		padding: 0 $large-content-padding;
+		padding: 0 $large-content-padding $large-content-padding $large-content-padding;
```

Comment:
This is slightly more compacted form, but your suggestion is much more clear for sure. There are other instances that can be changed in this file. What's your take @lizlux ?